quantum/quantum.c

```diff
@@ -357,6 +357,26 @@ bool process_record_quantum(keyrecord_t *record) {
             case ONESHOT_DISABLE:
                 oneshot_disable();
                 break;
+#endif
+#ifdef ENABLE_COMPILE_KEYCODE
+            case QK_MAKE: // Compiles the firmware, and adds the flash command based on keyboard bootloader
+            {
+#    ifdef NO_ACTION_ONESHOT
+                const uint8_t temp_mod = mod_config(get_mods());
+#    else
+                const uint8_t temp_mod = mod_config(get_mods() | get_oneshot_mods());
+                clear_oneshot_mods();
+#    endif
+                clear_mods();
+
+                send_string_with_delay_P(PSTR("qmk"), TAP_CODE_DELAY);
```

Comment:
Any reason this one was different to the ones below?
```suggestion
                SEND_STRING_DELAY("qmk", TAP_CODE_DELAY);
```

Reply:
Nope.  Just habit.